README.md

```diff
@@ -20,10 +20,20 @@ Godel: Austrian logician and mathematician http://en.wikipedia.org/wiki/Kurt_G%C
   cd ..
   catkin_make
   ```
+  
+### Application can work at two modes:
+-  Local mode
+  -  All the ROS nodes run at the same machine.
+-  Network mode
+  -  ROS nodes run at two different machine: Server and Client.
+  -  Client runs ROS nodes "RVIZ" and "point_cloud_generator_node".
+  -  Server runs all other ROS nodes except two nodes running on client.
```

Comment:
Besides move_group node, the **simulator_service_node** is also trying to connect "Joint_trajectory_action" robot action server. 

Your suggestion is that we put **Joint_trajectory_action** and **industrial_robot_simulator** on client side, and run client first,  then run server?

How about another solution to let **Joint_trajectory_action** working at server side and **industrial_robot_simulator** working at client side?  If you have any idea about why we see 40 seconds delay ?